client/ayon_blender/plugins/publish/validate_deadline_publish.py

```diff
@@ -41,21 +57,30 @@ def process(self, instance):
                 output_node = node
                 break
         if not output_node:
-            raise PublishValidationError(
-                "No output node found in the compositor tree."
-            )
+            msg = "No output node found in the compositor tree."
+            invalid.append(msg)
+
         filepath = bpy.data.filepath
         file = os.path.basename(filepath)
         filename, ext = os.path.splitext(file)
         if filename not in output_node.base_path:
-            raise PublishValidationError(
+            msg = (
                 "Render output folder doesn't match the blender scene name! "
                 "Use Repair action to fix the folder file path."
             )
+            invalid.append(msg)
+        if not bpy.context.scene.render.filepath:
+            msg = (
+                "No render filepath set in the scene!"
+                "Use Repair action to fix the render filepath."
+            )
+            invalid.append(msg)
+        return invalid
 
     @classmethod
     def repair(cls, instance):
         container = instance.data["transientData"]["instance_node"]
         prepare_rendering(container)
         bpy.ops.wm.save_as_mainfile(filepath=bpy.data.filepath)
+        bpy.context.scene.render.filepath = "/tmp\\"
```

Comment:
Why do we need the double backslash at the end? That won't work on UNIX devices? This may need to be:
```suggestion
        bpy.context.scene.render.filepath = "/tmp/"
```